#! /usr/bin/env python

PACKAGE='draco_point_cloud_transport'

from dynamic_reconfigure.parameter_generator_catkin import *

gen = ParameterGenerator()

gen.add("SkipDequantizationPOSITION", bool_t,   0, "Tells decoder to skip dequantization of POSITION attributes",  False)
gen.add("SkipDequantizationNORMAL", bool_t,   0, "Tells decoder to skip dequantization of NORMAL attributes",  False)
gen.add("SkipDequantizationCOLOR", bool_t,   0, "Tells decoder to skip dequantization of COLOR attributes",  False)
gen.add("SkipDequantizationTEX_COORD", bool_t,   0, "Tells decoder to skip dequantization of TEX_COORD attributes",  False)
gen.add("SkipDequantizationGENERIC", bool_t,   0, "Tells decoder to skip dequantization of GENERIC attributes",  False)

exit(gen.generate(PACKAGE, "DracoSubscriber", "DracoSubscriber"))
